PatternPal/PatternPal.Core/StepByStep/Resources/Instructions/AdapterInstructions.resx

```diff
@@ -0,0 +1,144 @@
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Explanation1" xml:space="preserve">
+    <value>This interface defines some sort of behavior that we will want to adapt a class to.</value>
+  </data>
+  <data name="Explanation2" xml:space="preserve">
+    <value>The client cannot use this (possibly third party) class directly because it has an incompatable interface.</value>
+  </data>
+  <data name="Explanation3" xml:space="preserve">
+    <value>To be able to use the Service, we make this class to adapt it to our interface. When called, it "translates" the call to the Service. Its field is private, since the logic should be exposed to the other code as the general interface only. The real implementation of the Service should only be known by the adapter. </value>
+  </data>
+  <data name="Explanation4" xml:space="preserve">
+    <value>The client can now use the logic of the Service as if it does have a compatable interface.</value>
+  </data>
+  <data name="Step1" xml:space="preserve">
+    <value>Make an interface with a method. We refer to this interface as `ClientInterface`.</value>
+  </data>
+  <data name="Step2" xml:space="preserve">
+    <value>Make a class that has a method. It may not inherit from ClientInterface. We refer to this class as `Service`.</value>
+  </data>
+  <data name="Step3" xml:space="preserve">
+    <value>Make a class that implements ClientInterface. Give it a private field of type Service and instantiate it. Implement the method of ClientInterface by using the method of the field. We refer to this class as `Adapter`.</value>
+  </data>
+  <data name="Step4" xml:space="preserve">
+    <value>Make a class that creates an instance of Adapter. Give it a method that calls the method of the instantiated variable.</value>
```

Comment:
Om deze stap te halen tijdens testen maakte ik een private field van type "Adapter" met = new() erachter, zoals ook in step 3. Moet hierbij nog geschreven worden dat er een field gemaakt moet worden?